andela/travel_tool_front
# #161580083 Send Deployment Failure Notification to Slack

#### What does this PR do?
- Sends notification to slack when any of the CirlceCI job fails on develop or master branch.

#### Description of Task to be completed?
Currently, when a deployment fails, CircleCI runs the entire deployment all over again. Fix this issue so that when a deployment fails, a failure notification is sent to slack.

#### How should this be manually tested?
Check for a notification on travela-deploy when any of the CircleCI deployment job fails.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/161580083
https://www.pivotaltracker.com/story/show/161580081

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
